cli/src/main/kotlin/commands/ReporterCommand.kt

```diff
@@ -54,7 +54,7 @@ object ReporterCommand : CommandWithHelp() {
     }
 
     @Parameter(description = "The ORT result file to use. Must contain a scan result.",
-            names = ["--ort-result-file", "-i"],
+            names = ["--ort-file", "-i"],
```

Comment:
Did you consider naming this '--scan-result-file' ?

Reply:
maybe: --read-scan-result

Reply:
I did consider `--scan-result-file` but did not want to use it for the same reason not to use `--analyzer-result-file`: We should not create the false impression that there would be such a thing as a dedicated scan and / or analyzer result file. These are all ORT result files (from their contents) that just happen to contain scan / analyzer results, respectively.